zephyr/CMakeLists.txt

```diff
@@ -1,37 +1,38 @@
 cmake_minimum_required(VERSION 3.5)
 
 if (CONFIG_ESP_SERIAL_FLASHER)
-    zephyr_include_directories(
-        "${ZEPHYR_CURRENT_MODULE_DIR}/include"
-        "${ZEPHYR_CURRENT_MODULE_DIR}/port"
-        "${ZEPHYR_CURRENT_MODULE_DIR}/private_include"
-    )
 
-    zephyr_interface_library_named(esp_flasher)
+zephyr_interface_library_named(esp_flasher)
```

Comment:
Lots of lines have been just reformatted to remove indentation, can we revert these?

Reply:
I don't have a strong opinion on CMake formatting, but it would be more consistent with the rest of `esp-serial-flasher` to leave it as is, and also make the diffs smaller and git blame more meaningful.